schema/mutation.js

```diff
@@ -320,10 +345,17 @@ const Mutation = new GraphQLObjectType({
       },
       resolve(parent, args) {
         console.log(args);
+        const credentialHash = web3.utils.sha3(JSON.stringify(args));
+        const data = contract.methods.addCredential(credentialHash).encodeABI();
         return Credential.insert(args)
-          .then(res => {
-            console.log(res);
+          .then(
+            res => {
             if (res) {
+              transaxFunc(data, function(receipt){
```

Comment:
It looks like we need a separate method for validating and invalidating credentials, since this is "updating" any other information. Since we're essentially issuing a new credential, should we be invalidating the old one on update?